schema/validator.go

```diff
@@ -46,17 +47,18 @@ func (e ValidationError) Error() string {
 }
 
 // Validate validates the given reader against the schema of the wrapped media type.
-func (v Validator) Validate(src io.Reader) error {
+// strict: accept the customized media type when value is false, or reject when true.
```

Comment:
@wking I think your words may be good as help message for customer, but I really want to present the design purpose here, as my words.